### ISSUE TITLE
After updating to macOS Ventura 13, I am not able to SSH any AWS EC2 server using this plugin 🧙‍♀️

### ISSUE THREAD

**Issue by mugiwarafx** [microsoft/vscode-remote-release#7481]
- VSCode Version: Version: 1.73.0 (Universal)
Commit: 8fa188b2b301d36553cbc9ce1b0a146ccb93351f
- Local OS Version: macOS Ventura Version 13.0 (22A380)
- Remote OS Version: Ubuntu 22.04.1 LTS (GNU/Linux 5.15.0-1022-aws x86_64)
- Remote Extension/Connection Type: AWS EC2 OpenSSH_8.9p1 Ubuntu-3, OpenSSL 3.0.2 15 Mar 2022
- Logs:
```log
[16:45:28.684] Log Level: 2
[16:45:28.686] remote-ssh@0.91.2022103115
[16:45:28.686] darwin x64
[16:45:28.692] SSH Resolver called for "ssh-remote+IP.eu-west-3.compute.amazonaws.com", attempt 1
[16:45:28.693] "remote.SSH.useLocalServer": true
[16:45:28.693] "remote.SSH.path": undefined
[16:45:28.693] "remote.SSH.configFile": undefined
[16:45:28.694] "remote.SSH.useFlock": true
[16:45:28.694] "remote.SSH.lockfilesInTmp": false
[16:45:28.694] "remote.SSH.localServerDownload": auto
[16:45:28.695] "remote.SSH.remoteServerListenOnSocket": false
[16:45:28.695] "remote.SSH.showLoginTerminal": true
[16:45:28.695] "remote.SSH.defaultExtensions": []
[16:45:28.695] "remote.SSH.loglevel": 2
[16:45:28.695] "remote.SSH.enableDynamicForwarding": true
[16:45:28.696] "remote.SSH.enableRemoteCommand": false
[16:45:28.696] "remote.SSH.serverPickPortsFromRange": {}
[16:45:28.696] "remote.SSH.serverInstallPath": {}
[16:45:28.699] SSH Resolver called for host: IP.eu-west-3.compute.amazonaws.com
[16:45:28.700] Setting up SSH remote "IP.eu-west-3.compute.amazonaws.com"
[16:45:28.702] Acquiring local install lock: /var/folders/lt/rksts1mn7p7f78615mlf9g7h0000gn/T/vscode-remote-ssh-c41bc994-install.lock
[16:45:28.704] Looking for existing server data file at /Users/username/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-c41bc994-8fa188b2b301d36553cbc9ce1b0a146ccb93351f-0.91.2022103115/data.json
[16:45:28.704] Using commit id "8fa188b2b301d36553cbc9ce1b0a146ccb93351f" and quality "stable" for server
[16:45:28.709] Install and start server if needed
[16:45:28.711] PATH: /usr/local/opt/mysql-client/bin:/opt/local/bin:/opt/local/sbin:/Users/username/.fastlane/bin:/usr/local/lib/ruby/gems/3.1.0/bin:/usr/local/opt/ruby/bin:/Users/username/.npm-global/bin:/Users/username/.pyenv/shims:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/Library/Apple/usr/bin:/Users/username/bin:/Users/username/Library/Python/3.9/bin:/Users/username/.local/bin
[16:45:28.711] Checking ssh with "ssh -V"
[16:45:28.718] > OpenSSH_9.0p1, LibreSSL 3.3.6

[16:45:28.720] askpass server listening on /var/folders/lt/rksts1mn7p7f78615mlf9g7h0000gn/T/vscode-ssh-askpass-ad7f36e3be8d7bb686449691f17a9b86f7aa9b5c.sock
[16:45:28.721] Spawning local server with {"serverId":1,"ipcHandlePath":"/var/folders/lt/rksts1mn7p7f78615mlf9g7h0000gn/T/vscode-ssh-askpass-7511035ae3380ce5121b4ced1447b59266542420.sock","sshCommand":"ssh","sshArgs":["-v","-T","-D","55985","-o","ConnectTimeout=15","IP.eu-west-3.compute.amazonaws.com"],"serverDataFolderName":".vscode-server","dataFilePath":"/Users/username/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-c41bc994-8fa188b2b301d36553cbc9ce1b0a146ccb93351f-0.91.2022103115/data.json"}
[16:45:28.721] Local server env: {"SSH_AUTH_SOCK":"/private/tmp/com.apple.launchd.z7qeAIkcxb/Listeners","SHELL":"/usr/local/bin/bash","DISPLAY":"1","ELECTRON_RUN_AS_NODE":"1","SSH_ASKPASS":"/Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.91.2022103115/out/local-server/askpass.sh","VSCODE_SSH_ASKPASS_NODE":"/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper.app/Contents/MacOS/Code Helper","VSCODE_SSH_ASKPASS_EXTRA_ARGS":"--ms-enable-electron-run-as-node","VSCODE_SSH_ASKPASS_MAIN":"/Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.91.2022103115/out/askpass-main.js","VSCODE_SSH_ASKPASS_HANDLE":"/var/folders/lt/rksts1mn7p7f78615mlf9g7h0000gn/T/vscode-ssh-askpass-ad7f36e3be8d7bb686449691f17a9b86f7aa9b5c.sock"}
[16:45:28.722] Spawned 93977
[16:45:28.859] > local-server-1> Spawned ssh, pid=93995
[16:45:28.862] stderr> OpenSSH_9.0p1, LibreSSL 3.3.6
[16:45:43.897] stderr> ssh: connect to host IP.eu-west-3.compute.amazonaws.com port 22: Operation timed out
[16:45:43.897] > local-server-1> ssh child died, shutting down
[16:45:43.901] Local server exit: 0
[16:45:43.902] Received install output: local-server-1> Spawned ssh, pid=93995
OpenSSH_9.0p1, LibreSSL 3.3.6
ssh: connect to host IP.eu-west-3.compute.amazonaws.com port 22: Operation timed out
local-server-1> ssh child died, shutting down

[16:45:43.904] Resolver error: Error: The operation timed out
	at g.Offline (/Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.91.2022103115/out/extension.js:1:583872)
	at /Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.91.2022103115/out/extension.js:1:581509
	at t.handleInstallOutput (/Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.91.2022103115/out/extension.js:1:581661)
	at Object.e [as tryInstallWithLocalServer] (/Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.91.2022103115/out/extension.js:1:623084)
	at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
	at async /Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.91.2022103115/out/extension.js:1:642212
	at async t.withShowDetailsEvent (/Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.91.2022103115/out/extension.js:1:645932)
	at async /Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.91.2022103115/out/extension.js:1:621556
	at async T (/Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.91.2022103115/out/extension.js:1:618062)
	at async t.resolveWithLocalServer (/Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.91.2022103115/out/extension.js:1:621171)
	at async t.resolve (/Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.91.2022103115/out/extension.js:1:643540)
	at async /Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.91.2022103115/out/extension.js:1:722351
[16:45:43.911] ------




[16:45:44.018] ------




[16:45:44.019] SSH Resolver called for "ssh-remote+IP.eu-west-3.compute.amazonaws.com", attempt 2, (Reconnection)
[16:45:44.021] SSH Resolver called for host: IP.eu-west-3.compute.amazonaws.com
[16:45:44.021] Setting up SSH remote "IP.eu-west-3.compute.amazonaws.com"
[16:45:44.022] Acquiring local install lock: /var/folders/lt/rksts1mn7p7f78615mlf9g7h0000gn/T/vscode-remote-ssh-c41bc994-install.lock
[16:45:44.022] Looking for existing server data file at /Users/username/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-c41bc994-8fa188b2b301d36553cbc9ce1b0a146ccb93351f-0.91.2022103115/data.json
[16:45:44.023] Using commit id "8fa188b2b301d36553cbc9ce1b0a146ccb93351f" and quality "stable" for server
[16:45:44.027] Install and start server if needed
[16:45:44.031] askpass server listening on /var/folders/lt/rksts1mn7p7f78615mlf9g7h0000gn/T/vscode-ssh-askpass-b9a61a37b2d0cb89096f6ac77fd100234b4690d5.sock
[16:45:44.031] Spawning local server with {"serverId":2,"ipcHandlePath":"/var/folders/lt/rksts1mn7p7f78615mlf9g7h0000gn/T/vscode-ssh-askpass-d4985616c104b8a7860e51b23a78afc9d3a9941a.sock","sshCommand":"ssh","sshArgs":["-v","-T","-D","55990","-o","ConnectTimeout=15","IP.eu-west-3.compute.amazonaws.com"],"serverDataFolderName":".vscode-server","dataFilePath":"/Users/username/Library/Application Support/Code/User/globalStorage/ms-vscode-remote.remote-ssh/vscode-ssh-host-c41bc994-8fa188b2b301d36553cbc9ce1b0a146ccb93351f-0.91.2022103115/data.json"}
[16:45:44.032] Local server env: {"SSH_AUTH_SOCK":"/private/tmp/com.apple.launchd.z7qeAIkcxb/Listeners","SHELL":"/usr/local/bin/bash","DISPLAY":"1","ELECTRON_RUN_AS_NODE":"1","SSH_ASKPASS":"/Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.91.2022103115/out/local-server/askpass.sh","VSCODE_SSH_ASKPASS_NODE":"/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper.app/Contents/MacOS/Code Helper","VSCODE_SSH_ASKPASS_EXTRA_ARGS":"--ms-enable-electron-run-as-node","VSCODE_SSH_ASKPASS_MAIN":"/Users/username/.vscode/extensions/ms-vscode-remote.remote-ssh-0.91.2022103115/out/askpass-main.js","VSCODE_SSH_ASKPASS_HANDLE":"/var/folders/lt/rksts1mn7p7f78615mlf9g7h0000gn/T/vscode-ssh-askpass-b9a61a37b2d0cb89096f6ac77fd100234b4690d5.sock"}
[16:45:44.033] Spawned 94867
[16:45:44.162] > local-server-2> Spawned ssh, pid=94884
[16:45:44.164] stderr> OpenSSH_9.0p1, LibreSSL 3.3.6
```

Steps to Reproduce:

1. Press F1 > Remote-SHH: Connect to Host...
2. ssh user@host -i ~/.ssh/key.pem

If I copy-paste the step2 command on the terminal, works flawlessly.

Does this issue occur when you try this locally?: No
Does this issue occur when you try this locally and all extensions are disabled?: Yes

**Comment by eleanorjboyd** [comment]
Hi! Thank you for writing such a verbose issue- we really appreciate it! I think your problem might be connected to this ongoing issue: https://github.com/microsoft/vscode-remote-release/issues/7324. Does that seem correct?

**Comment by mugiwarafx** [comment]
Hi @eleanorjboyd! 

First of all, sorry for delaying my response.

Secondly, I was wrong. The plugin works flawlessly. The problem was with the new openSSH version; for some reason (I have no clue which), I had to modify my SSH client config to make it work again.

So the proper title for this issue should be: [Since I updated to Ventura, I cannot SSH to my server (Ubuntu 20.04) from my macOS](https://superuser.com/questions/1752882/since-i-updated-to-ventura-i-cannot-ssh-to-my-server-ubuntu-20-04-from-my-mac)

But again, it has nothing to do with your plugin 🙂

Let me know if I should delete the issue or if you prefer that I keep this conversation logged here just in case it can help someone.
